### DISCUSSION TITLE
【求助】下载文件时，显示访问 public.dm.files.1drv.com 被拒绝

### DISCUSSION THREAD

**Discussion by wangoho** [spencerwooo/onedrive-vercel-index#525] [Q&A]
这是文件地址：（其他文件下载正常）
https://dir.hooe.cc/api/raw/?path=/%E5%AE%89%E8%A3%85%E5%8C%85/HEU_KMS_Activator_v24.6.1.zip

![image](https://user-images.githubusercontent.com/75483499/156931106-62ebcd67-9673-4295-aeed-e088e095ac08.png)

**Comment by Android-KitKat** [comment]
请检查文件是否被标记为恶意软件而受到限制。
如果是非个人版，可由管理员设置取消阻止。
[在 SharePoint Online、OneDrive 或 Microsoft Teams 中发现恶意文件时需采取的操作](https://support.office.com/article/01e902ad-a903-4e0f-b093-1e1ac0c37ad2)
